tests/libYARP_OS/ValueTest.cpp

```diff
@@ -235,6 +235,11 @@ class ValueTest : public UnitTest {
             checkTrue((v1!=v2), "(blob) operator!= ok");
             checkTrue((v1!=*(reinterpret_cast<const int*>(v2.asBlob()))), "(blob) value ok");
         }
+
+        {
+            Value v(0);
+            checkTrue(v == 0, "equivalence with integer literal ok");
```

Comment:
Unclear to me: what we want to test here?
According to the lately introduced `YARP_DEPRECATED_MSG`, shouldn't this be `v.asInt() == 0`?